src/sass/partials/global/_mixins.scss

```diff
@@ -62,11 +62,11 @@
       border: setting-get('input border width') solid setting-get('primary color');
       content: '';
       display: inline-block;
-      height: 1em;
-      margin-right: setting-get('rhythm') / 2;
-      transform: translateY(-.125em);
+      height: 1.25em;
+      margin-right: setting-get('rhythm') / 4;
+      transform: translateX(-.8rem);
```

Comment:
* You're mixing `rem` and `em` for translation in `checkbox`. You should use one or the other
* Where does the `.8` come from? That seems like a slightly arbitrary number